pkg/skaffold/schema/latest/config.go

```diff
@@ -279,6 +282,22 @@ type DockerConfig struct {
 	SecretName string `yaml:"secretName,omitempty"`
 }
 
+// ResourceRequirements describes the resource requirements for the kaniko pod
+type ResourceRequirements struct {
+	// Requests https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container
```

Comment:
Can you add a short description and use Mardown notation for the link?

Reply:
done